.github/workflows/ci-operator.yaml

```diff
@@ -0,0 +1,37 @@
+name: Operator Installation Test
+on:
+  pull_request:
+    paths:
+      - 'operator/**'
```

Comment:
Does it require more effort?

```yaml
  push:
    paths:
      - 'operator/**'
  pull_request:
   paths:
      - 'operator/**'
# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency
concurrency:
  group: ${{ github.workflow }}-${{ github.actor }}-${{ github.head_ref || github.run_id }}
  cancel-in-progress: true
```

Reply:
Hi~ @liangyuanpeng  Since this ci is triggered when the pr is submitted, and the pr can only be merged in if the ci is successful, I don't think this ci needs to be triggered again on push.